magma/wire_container.py

```diff
@@ -1,6 +1,13 @@
+import logging
+from magma.config import config, EnvConfig
 from magma.logging import root_logger, StagedLogRecord
 
 
+config._register(
+    rewire_log_level=EnvConfig("MAGMA_REWIRE_LOG_LEVEL", "WARN")
```

Comment:
"WARN" is deprecated, I think "WARNING" is preferred now

```suggestion
    rewire_log_level=EnvConfig("MAGMA_REWIRE_LOG_LEVEL", "WARNING")
```